src/network/networkd-network-gperf.gperf

```diff
@@ -271,6 +274,7 @@ DHCPv6.DUIDRawData,                          config_parse_duid_rawdata,
 DHCPv6.RapidCommit,                          config_parse_bool,                                        0,                             offsetof(Network, dhcp6_use_rapid_commit)
 DHCPv6.NetLabel,                             config_parse_string,                                      CONFIG_PARSE_STRING_SAFE,      offsetof(Network, dhcp6_netlabel)
 DHCPv6.SendRelease,                          config_parse_bool,                                        0,                             offsetof(Network, dhcp6_send_release)
+DHCPv6.NFTSet,                               config_parse_dhcp6_nft_set_context,                       0,                             0
```

Comment:
this should just use config_parse_nft_set_context() and then use offsetof() in the rightmost colun to point the call to the right field.

Reply:
We need two offsets, one for the set context and other for the counter. It's possible to calculate the delta offset between those and add that to the one and only offset, but that would involve some pointer arithmetic. Perhaps a  struct containing both context and counter could be used to avoid that.

Reply:
Now using the struct approach. For Address.IPv{4,6}NFTSet this isn't possible since the pointer passed to the function seems to be to a Network and not Address.